src/06-loops.js

```diff
@@ -61,7 +133,18 @@ function rangeEveryOther(min, max) {}
  *  containsWhileLoop([ "left", "up", "right" ], "down");
  *  //> false
  */
-function containsWhileLoop(array, target) {}
+
+function containsWhileLoop(array, target) {
+  let i = 0
+    while (i < array.length) {
+      if (array[i] === target) {
+        return true
+      }
+      i++
+    }
+    return false
+}
+
```

Comment:
Great job on the `contains` functions. Returning early as soon as you have an answer is a pro move, right there!